mantidproject/mantid
# Indirect - Replace old MsdYi.py file with updated version

**Description of work.**
This PR replaces the old version of the file MsdYi.py file with the updated version. In the previous version the expression for the function is incorrect.

**To test:**
1.`Interfaces`->`Indirect`->`Data Analysis`->`MSDFit` Tab
2. Load the data below
3. Fit Type: `Yi`
4. X Range: 0 to 0.9
5. Click `Run` and wait

- The process should run without any failures.

**Data Files**
[osi104367-104371_graphite002_red_elwin_eq.zip](https://github.com/mantidproject/mantid/files/2530414/osi104367-104371_graphite002_red_elwin_eq.zip)

Fixes #23776 

~~No release notes required.~~

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
